tests/boundary_condition_test.py

```diff
@@ -9,19 +10,16 @@
 def test_outdoors():
     bc = bcs.outdoors
     assert bc.name == 'Outdoors'
-    assert bc.sun_exposure == True
+    assert bc.sun_exposure is True
```

Comment:
Already sent you a link about this. `==` is not the best but is better than using `is` for checking equality for boolean values.

Reply:
Right!  I just fixed it.